packages/react/src/components/Card/Card.module.css

```diff
@@ -0,0 +1,154 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  /*
+   * TODO: discuss how we would like to determine max width
+   */
+  width: 330px;
+}
```

Comment:
This needs an `overflow: hidden`, otherwise the image will go outside the border radius set.